docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,134 @@
+        #. You MUST also set this subsection to be Graded as a Midterm or Final exam
+        #. This subsection should contain at least one unit with one gradeable block, e.g. a generic multiple choice question in order to test downstream effects to the gradebook
+    #. You should also have a tab open with your event bus logs
+
+
+Proctored Exam Flow
+-------------------
+Expected Behavior: "A paid track learner is able to start, complete, and submit a proctored exam using Proctoring Software that is launched via LTI."
+
+Can't Start Exam Without Proctoring Software Installed
+======================================================
+#. Enter a Proctored Exam without proctoring software installed. You should see an interstitial stating that "This exam is proctored" with a button that says "Continue to Proctored Exam"
+#. Clicking "Continue to Proctored Exam" should bring you to an interstitial titled "Set up and start your proctored exam." with "Start System Check" and "Start Exam" buttons
+#. Clicking the "Start Exam" button should pop up a modal that says "Cannot Start Exam"
+#. Clicking "OK" should close that modal
+#. Clicking "Start System Check" should redirect you to your proctoring setup page
+#. Navigating back one page and clicking "Start Exam" should still block you from starting the exam
+
+Can't Start Exam Without Proctoring Software Running
+====================================================
+#. Install the proctoring software that you've configured with your platform. Go through the opening interstitials and click "Start System Check". This should load your respective Proctoring Software Setup page
+#. Immediately navigating back and trying to click "Start Exam" should't allow you to start the exam
+#. Navigating back to your proctoring software setup page and refreshing on that page should cause no issues
+#. Go through your proctoring software setup for your Webcam, Desktop, etc. Upon setup completion, you should be redirected back to an exam interstitial with just the "Start Exam" button
+#. Close your proctoring software and click "Start Exam". The exam should not start
+
+Exam Flow Works With Proctoring Software Installed & Running
+============================================================
+#. Go through the setup again, this time without closing your proctoring software. Clicking "Start Exam" should start the exam
+#. Complete whatever question you put inside your exam w/ the correct answer. Clicking "End My Exam" should direct you to a "Are you sure you want to end your proctored exam?" interstitial
+#. Clicking "No I'd like to continue working" should send you back to the exam
+#. Clicking "Yes end my proctored exam" should end the exam and directs you to a "You have submitted this proctored exam for review" page
+#. In the course view, the exam you took should be marked as completed (with a green checkmark)
+
+
```

Comment:
Are there any tests for what to expect when an error occurs? For the proctoring flow, these all look like "happy path" tests.

Reply:
Yes those should be here: https://github.com/edx/edx-exams/pull/205/files/d1455d82318fcaebb675be2eff36909ceee74c6e#diff-b5c1a36768f79123cc3158c9c56b1b72e50008252b380bec3d570b0a2ca0cc37R119. I'm realizing I need to revise these to say "As a student, error the exam, and as an admin, reset the exam attempt, then try the exam again as a student" so that we reflect what would happen IRL.

Reply:
I see we're testing the exams dashboard for error attempts but to Alie's point shouldn't we have a test of an error from the "testing the exam flow" perspective? Something to test that the learner gets an error interstitial / the learner cannot access the exam content after an error.